### ISSUE TITLE
Word export or copy - arrows

### ISSUE THREAD

**Issue by berkdemir** [Proektsoftbg/Calcpad#256]
Hi,
Weird arrows appear when I copy the results to word. For me, Calcpad is perfect, but I still need to use screenshot to include the calculations in my report. The copy and paste with "merge formatting" works almost good, but there are additional shapes coming up which is not seen in Calcpad.

![image](https://github.com/Proektsoftbg/Calcpad/assets/40423004/fa502cc5-5602-4a34-a024-e35bf58ffbb7)
![image](https://github.com/Proektsoftbg/Calcpad/assets/40423004/77d8ed7b-2dd8-4b9a-8bba-b5ca437d294b)

**Comment by Proektsoftbg** [comment]
Hi!

What version do you use? It does not happen with version 6.2.0. At least I cannot reproduce it.

**Comment by ExiMentor** [comment]
That happens to me, too. I just updated to 6.2.0 and opend a file to test.

I think these are the arrows that appear on the left, when you hover over with the mouse.
![grafik](https://github.com/Proektsoftbg/Calcpad/assets/147033285/933e89ba-dbd7-4a49-8c43-190980fc8c28)

![grafik](https://github.com/Proektsoftbg/Calcpad/assets/147033285/ad507aa2-198c-41be-80b2-468265e59e41)

**Comment by Proektsoftbg** [comment]
Hmm. Maybe it depends on the version of Word. I use Office 365.

Yes, these are exactly those arrows. This option was requested previously to find the respective lines in the source code from the output. For this purpose, they are present in the Html and when when you select and copy, they are copied too.

In general avoid copying the Html output directly to Word. Use the built-in export instead. The export function filters out all the unnecessary stuff and convert formulas to MathtType.

If you prefer coping, first go to input form mode (code on the left not visible). Arrows are not present in this mode.
Finally, you can use find-replace command in Word to clear these symbols in the whole document at once.

**Comment by ExiMentor** [comment]
I have Office 365, too. Maybe you are not pasting with this option:
![grafik](https://github.com/Proektsoftbg/Calcpad/assets/147033285/6675d69a-e5bc-41a1-8788-af007f34c0ec)


I prefer the export, too. Anyway, it is sometimes faster to just copy what you need.

Maybe you can find another solution for the arrows in the future.

**Comment by Proektsoftbg** [comment]
No, I just paste it. Do you select and copy or use the copy button above?

Arrows work so fine and beautiful and I do not want to remove them. Maybe in the future we will even need to add some more items to make the output more interactive.

I think it is possible to detect when you copy the output and do some cleanup in the clipboard. But until then, please use the above instructions.

**Comment by ExiMentor** [comment]
I was using ctrl + c. To be honest, I never saw the button before.

The result is even worst since there is another symbol coming (you can see that at berkdemirs screenshot as well):
![grafik](https://github.com/Proektsoftbg/Calcpad/assets/147033285/097770ab-69f9-4d24-aa8e-37a7bb12b72e)

I like the arrows, too. But they are also a bit fishy to catch. If it would be possible, that those are indipendtly from the output box.
I don't know how you programmed that, but if there would be an extra row with those arrows, It might be possible to prevent copying them.
Like one row with the actual output and another row with controlles or indicators like the arrows.

**Comment by Proektsoftbg** [comment]
OK, thank you. I see.

We will add some cleanup that will trigger on the copy command to remove the arrows and other stuff. This will be the best solution.
